OutaTime/OutaTime/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  OutaTime
+//
+//  Created by Keri Levesque on 1/29/20.
+//  Copyright © 2020 Keri Levesque. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+
+
+}
+
```

Comment:
If you are not using this file in your project make sure to delete it before pushing.